src/devices/src/virtio/block/device.rs

```diff
@@ -484,9 +489,69 @@ pub(crate) mod tests {
             &mut EventManager::new().unwrap(),
         );
         // Validate the queue operation finished successfully.
+        #[cfg(not(fuzzing))]
         assert_eq!(b.interrupt_evt.read().unwrap(), 1);
     }
 
+    #[cfg(fuzzing)]
+    pub fn fuzzing_func(data: &[u8]) {
+        const DESC_DATA_SIZE: usize = 5;
+        const VQUEUE_SIZE: usize = 16;
+        let mut block = default_block();
+        let mem = default_mem();
+        let vq = VirtQueue::new(GuestAddress(0), &mem, VQUEUE_SIZE as u16);
+        let mut idx = 0;
+        let mut data_size = data.len();
```

Comment:
I'd add a comment here saying "Processing AFL offered data, this data will be split in u8 and populate the fields of a Virtio descriptor.